includes/integrations/software-licensing.php

```diff
@@ -44,4 +53,24 @@ private function has_license_check( $status = '' ) {
 		return false;
 	}
 
+	private function has_all_license_check( $status = '' ) {
```

Comment:
```suggestion
	private function has_all_license_check( $status ) {
```

Because there's no use case for `$status` being empty, let's make it required.

With this change, the `empty()` check immediately below can then be removed.

Reply:
Done!